docs/api/plugins.md

```diff
@@ -439,6 +446,24 @@ uniform request distribution.  To enable this, you can pass in
 `options.tokensTable`, which is simply any Object that supports `put` and `get`
 with a `String` key, and an `Object` value.
 
+## Inflight Request Throttling
+
+```js
+const options = { limit: 600, server: server };
```

Comment:
The `server` property is not in docs. Do we need to pass in `server` or should we be only passing in the specific properties that the throttle plugin needs?